apps/docs/src/app/core/component-docs/popover/examples/popover-complex-example/popover-complex-example.component.html

```diff
@@ -6,7 +6,7 @@
         <fd-popover-body>
             <ul fd-list>
                 <li fd-list-item *ngFor="let option of list">
-                    <a [attr.href]="option.url" fd-list-title>{{ option.text }}</a>
+                    <a [attr.href]="option.url" fd-list-title tabindex="-1">{{ option.text }}</a>
```

Comment:
I think we need to find a way to do this automatically, so the dev doesn't have to enter this every time

Reply:
<img width="185" alt="Screenshot 2021-08-03 at 9 58 58 PM" src="https://user-images.githubusercontent.com/53534379/128052079-527aa83d-d14a-4502-86ef-a1dc07a3f9a9.png">

by default <a> tag gets focus, to over ride the focus we can add tabindex in the fd-link at the host, but the font color changes as shown in the screen shot.

current implementation 
<img width="185" alt="Screenshot 2021-08-03 at 10 02 37 PM" src="https://user-images.githubusercontent.com/53534379/128052619-fa818191-fab4-4143-82cc-767c4a2f0f8a.png">

Reply:
It could be done programmatically by drilling down through ContentChildren/ViewChildren and setting the tabindex manually.  A bit extreme but helps remove unnecessary code for the developer

Reply:
In NGX the developer should not add `tabindex="-1"`, this should be handled by fund-ngx behind the scene.